src/app/src/pages/Mainpage.tsx

```diff
@@ -85,9 +63,6 @@ export default function Mainpage({ currentUser }: Props) {
 
   return (
     <MainPageContainer>
-      <NavBarContainer>
```

Comment:
Can we do this for the other pages, or were you planning on that in your design PR?
<img width="1280" alt="Screen Shot 2022-04-03 at 2 24 13 PM" src="https://user-images.githubusercontent.com/55111012/161447013-981e17ae-5246-4f66-b8c7-1106d32e39e9.png">